src/lib/validate-input.js

```diff
@@ -611,6 +611,18 @@ export function validateGatherParams(input, indices, {axis = 0} = {}) {
   }
 }
 
+export function validateCumulativeSumParams(input, axis) {
+  if (axis !== undefined) {
+    const rank = input.rank;
+    if (!Number.isInteger(axis) || axis < -rank || axis >= rank) {
```

Comment:
The axis is of `unsigned long`, so its range is in `[0, rank)`. Please modify this check and delete the tests with negative axis, thanks.

Reply:
done

Reply:
Yeah, we generally want WebNN to be more explicit, resolving these kinds of user-facing API conveniences (like negative numbers and output shape rounding) into lower-level concrete values, like the actual axis and specific output shape.